src/routes/guides/urls.ts

```diff
@@ -5,56 +5,55 @@ export const GUIDES = {
   BUILD_APP: {
     title: "Building an application",
     href: "/guides/building-an-application",
-    cardImageAlt: "Paper sheet",
+    cardImageAlt: "",
     cardImage: GuidesIcon,
   },
   BUILD_DATA_SOURCE: {
     title: "Building a data source",
     href: "/guides/building-a-data-source",
-    cardImageAlt: "Paper sheet",
+    cardImageAlt: "",
     cardImage: GuidesIcon,
   },
   BUILD_DATA_DEF: {
     title: "How to create data definitions",
     href: "/guides/building-a-data-definition",
-    cardImageAlt: "Paper sheet",
+    cardImageAlt: "",
     cardImage: GuidesIcon,
   },
   VERIFY_CONSENT: {
     title: "How to verify consent in data source",
     href: "/guides/verifying-consent-in-data-source",
-    cardImageAlt: "Paper sheet",
+    cardImageAlt: "",
     cardImage: GuidesIcon,
   },
   MANAGE_GROUP: {
     title: "How to create and manage a group",
     href: "/guides/managing-groups",
-    cardImageAlt: "Paper sheet",
+    cardImageAlt: "",
     cardImage: GuidesIcon,
   },
   MANAGE_GROUP_MEMBERS: {
     title: "How to manage group members",
     href: "/guides/managing-group-members",
-    alt: "Data schemas",
+    cardImageAlt: "",
     cardImage: GuidesIcon,
   },
   USING_LOGIN_PROVIDER: {
     title: "How to use dataspace login in an application",
     href: "/guides/using-login-provider",
-    cardImageAlt: "Paper sheet",
+    cardImageAlt: "",
     cardImage: GuidesIcon,
   },
   VERIFY_ID_TOKEN: {
     title: "Verify id_token in data source",
     href: "/guides/verifying-id-token",
-    cardImageAlt: "Paper sheet",
+    cardImageAlt: "",
```

Comment:
If you are removing the `alt`, you don't do it by just changing configuration, you change the implementation. Remove `cardImageAlt` completely from this map.

Also I don't see much point in repeating `GuidesIcon`, is there one?